_tools/default_data.py

```diff
@@ -0,0 +1,125 @@
+#
+# Default data, only used when RIOTBASE is not provided or doesn't exist
+#
+DEFAULT_BOARDS = [
+    { "name": "\"Arduino Duemilanove\"", "group": "arduino-duemilanove" },
+    { "name": "\"Atmel SAM D10 Xplained Mini\"", "group": "samd10-xmini" },
+    { "name": "\"MSB-IoT\"", "group": "msbiot" },
+    { "name": "\"Native Board\"", "group": "native" },
+    { "name": "\"Particle Xenon\"", "group": "particle-xenon" },
+    { "name": "\"PineTime\"", "group": "pinetime" },
+    { "name": "\"STM32 Nucleo-L4R5ZI\"", "group": "nucleo144-l4r5" },
+    { "name": "\"STM32 Nucleo-L552ZE-Q\"", "group": "nucleo-l552ze-q" },
+    { "name": "\"SiFive HiFive1 RISC-V board\"", "group": "hifive1" },
+    { "name": "\"nRF52840-MDK\"", "group": "nrf52840-mdk" }
```

Comment:
this is a lot of escaping, which makes it hard to read and error-prone when changing the stub data set.